modules/sox-workflowoperation/src/main/java/org/opencastproject/workflow/handler/sox/AnalyzeAudioWorkflowOperationHandler.java

```diff
@@ -118,43 +118,32 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, Jo
     logger.debug("Running analyze audio workflow operation on workflow {}", workflowInstance.getId());
 
     try {
-      return analyze(workflowInstance.getMediaPackage(), workflowInstance.getCurrentOperation());
+      return analyze(workflowInstance.getMediaPackage(), workflowInstance.getCurrentOperation(), workflowInstance);
```

Comment:
Why pass on the instance and one of its parameters?
```suggestion
      return analyze(workflowInstance.getMediaPackage(), workflowInstance);
```

Reply:
Yeah, I don't know what ran through my mind there. Changed.